corehq/apps/users/models.py

```diff
@@ -2374,12 +2377,40 @@ def is_member_of(self, domain_qs):
 #
 # Django  models go here
 #
-class Invitation(Document):
+class SQLInvitation(SyncSQLToCouchMixin, models.Model):
```

Comment:
Should this be an AbstractBaseClass: https://docs.djangoproject.com/en/1.8/topics/db/models/#abstract-base-classes? Just want to ensure you explicitly want multi-table inheritance, e.g. you'd like like a table for `SQLInvitation` and its subclasses that is linked via a one-to-one relation